### ISSUE TITLE
What loss function do you use?

### ISSUE THREAD

**Issue by zjuzuhe** [arnoweng/CheXNet#7]
Hello
Thx for your sharing code.I'm trying to make a caffe reimplementation of it.I use SigmoidCrossEntropyLoss ,and I modified batch size,learning rate several times.But I can't get a good result.Actually,I just get a mean AUROC of 70%.
Could you give some information of loss function about your work?

**Comment by arnoweng** [comment]
@zjuzuhe I have used [BCELoss](http://pytorch.org/docs/master/nn.html#torch.nn.BCELoss) in PyTorch.
